src/schema_change_handler.cpp

```diff
@@ -109,9 +109,8 @@ void SchemaChangeHandler::on_set(const ResponseMap& responses) {
       has_error = true;
     }
   }
-  if (has_error) return;
 
-  if (has_schema_agreement(responses)) {
+  if (!has_error && has_schema_agreement(responses)) {
```

Comment:
Not totally necessary, but it might be useful to log something more than `DEBUG`/schema not up-to-date if `has_error` (below, line 126).